openslides_backend/models/models.py

```diff
@@ -915,7 +923,8 @@ class Motion(Model):
     derived_motion_ids = fields.RelationListField(
         to={Collection("motion"): "origin_id"}
     )
-    forwarding_tree_motion_ids = fields.NumberArrayField()
+    all_origin_ids = fields.NumberArrayField()
```

Comment:
@jsangmeister These still have to be implemented, right?

Reply:
That is another pull request.

Reply:
right